service_kit/logging/git.py

```diff
@@ -0,0 +1,68 @@
+import subprocess
+
+from . import logger
+
+GIT = "/usr/bin/git"
+
+
+def _pwd_is_git_repository() -> bool:
+    try:
+        subprocess.run(
+            [GIT, "rev-parse", "--is-inside-work-tree"],
+            check=True,
+            stdout=subprocess.PIPE,
+            stderr=subprocess.PIPE,
+        )
+
+        return True
+    except subprocess.CalledProcessError:
+        return False
+
+
+def _get_commit_id():
+    process = subprocess.run(
+        [GIT, "rev-parse", "HEAD"], check=True, stdout=subprocess.PIPE, text=True
```

Comment:
What should be done in the case that the run raises an error (e.g., if the directory is not a git repo)?